packages/slate-editor/src/extensions/coverage/components/CoverageElement.tsx

```diff
@@ -35,8 +37,9 @@ export function CoverageElement({
     dateFormat,
     element,
     fetchCoverage,
+    onEdit,
 }: Props) {
-    const editor = useSlateStatic();
+    const editor = useSlate();
```

Comment:
Just curious what was the reason for this change?

Reply:
I saw that some components were using `useSlate` and some were using `useSlateStatic` and I wasn't sure what's the difference there, so I made it consistent with the `CoverageMenu`.

However I've checked the docs and I see that `useSlateStatic` does not trigger a re-render when there are changes in the editor, as oposed to `useSlate` which does.

To be honest, I'm not really sure which one is the correct one to use? 🤔 I guess it's better to use `useSlateStatic` in most cases to prevent re-renders?

Reply:
If short, you want to use `useSlate()` if your component depends on the editor state (like `children` or `selection`) -- to re-render it when the state changes, and `useSlateStatic()` when all you want is imperative API (to query and modify editor state _in response to user actions_).

Reply:
Ah, makes sense, thanks for explaining it 👍